eng/common/SetupNugetSources.ps1

```diff
@@ -1,167 +0,0 @@
-# This file is a temporary workaround for internal builds to be able to restore from private AzDO feeds.
```

Comment:
Oh, this is actually intentional -- this file needs to be deleted otherwise the SDL checks fail. Deleting it is currently the guidance from dnceng. There's an ongoing thread about a better solution.